Tasks/S3Upload/TaskOperations.ts

```diff
@@ -67,6 +67,23 @@ export class TaskOperations {
         }
     }
 
+    private getCacheControl(currentFile: string): string {
+        for (const expression of this.taskParameters.cacheControl) {
+            const firstIndex = expression.indexOf('=')
+            if (firstIndex < 1) {
+                continue
+            }
+            const matchStatemnt = expression.substring(0, firstIndex).trim()
+            const cacheControlSettings = expression.substring(firstIndex + 1).trim()
```

Comment:
If a cachecontrol entry was written as `*.ts=` without a value I believe this would blow up. You should make sure there is any data to substring and if not then log that cachcontrol entry is in an invalid format.